ui/src/submissions/conferences/containers/__tests__/ConferenceSubmissionSuccessPageContainer.test.jsx

```diff
@@ -1,43 +1,50 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
 import { Provider } from 'react-redux';
 import { MemoryRouter } from 'react-router-dom';
 import { fromJS } from 'immutable';
+import { render } from '@testing-library/react';
 
 import ConferenceSubmissionSuccessPageContainer, {
   ConferenceSubmissionSucessPage,
 } from '../ConferenceSubmissionSuccessPageContainer';
 import { getStoreWithState } from '../../../../fixtures/store';
 
 describe('ConferenceSubmissionSuccessPageContainer', () => {
+  const store = getStoreWithState({
```

Comment:
You can use `getStore` instead of `getStoreWithState` as we will deprecate the later one in the end of this initiative -> https://github.com/inspirehep/inspirehep/blob/master/ui/src/fixtures/store.js#L64-L70